build.sbt

```diff
@@ -39,7 +39,8 @@ lazy val warningSuppression = Seq(
   scalacOptions += "-Wconf:" + Seq(
     "msg=APIs in chisel3.internal:s",
     "msg=Importing from firrtl:s",
-    "msg=migration to the MLIR:s"
+    "msg=migration to the MLIR:s",
+    "msg=value Stream in package scala is deprecated:s"
```

Comment:
There are many lines like this
```
implicit val noShrinkListVal = Shrink[List[Boolean]](_ => Stream.empty)
```
I could not fund a way to replace Stream with LazyList because under 2.13 Shrink is parameterized with `Stream[T]`. Maybe we can talk about this. There's about 30 of the warnings without suppressing them.

Reply:
Will remove Shrink lines and the warningSuppression string